network/throttling/inbound_msg_byte_throttler.go

```diff
@@ -131,9 +133,19 @@ func (t *inboundMsgByteThrottler) Acquire(ctx context.Context, msgSize uint64, n
 	// Take as many bytes as we can from [nodeID]'s validator allocation.
 	// Calculate [nodeID]'s validator allocation size based on its weight
 	vdrAllocationSize := uint64(0)
-	weight := t.vdrs.GetWeight(nodeID)
+	weight := t.vdrs.GetWeight(t.subnetID, nodeID)
+	totalWeight, err := t.vdrs.TotalWeight(t.subnetID)
+	if err != nil {
+		t.log.Error("Failed to get total weight of network validators",
+			zap.Stringer("subnetID", t.subnetID),
+			zap.Error(err),
+		)
+		t.lock.Unlock()
+		return t.metrics.awaitingRelease.Dec
+	}
```

Comment:
I don't think we can just return here because we have already modified `t.nodeToAtLargeBytesUsed[nodeID]`

Reply:
moved it up